test/WebJobs.Script.Tests/Managment/FunctionsSyncManagerTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Script.WebHost.Management;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Managment
+{
+    public class FunctionsSyncManagerTests
```

Comment:
Why did you add a new file for this test - we already have an existing test suite here https://github.com/Azure/azure-functions-host/blob/6cef5808f90e3a9d3437209ab128361852120597/test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs#L31.

In the upcoming PR, let's fix this. Also, the test doesn't really test the feature/changes you added - ensuring that the actual sync triggers payload contains the expected content.

Reply:
Thanks @mathewc  - captured notes here: https://github.com/Azure/azure-functions-host/issues/7394